config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml

```diff
@@ -28,6 +28,7 @@ periodics:
   labels:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
+  run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|scripts|test|third_party|util)/|main\.go|go\.mod|Dockerfile|Makefile)'
```

Comment:
I'm not sure run_if_changed makes sense on a periodic job...
Also, should we increase the interval to 2h, given that the job takes ~40m? (it is almost continuously running)
/cc @vincepri ^^

Reply:
If we plan to add a post-submit job that runs all e2e tests, then makes sense to increase the interval. Otherwise, it would be good to get the signal sooner IMO.

Reply:
Having both postsubmit and periodic seems a duplication IMO. 
I'm for periodic only with 2h interval on the main branch, but this can be addressed also separately

WRT to this PR, we should probably remove run_if_changed for the period job

Reply:
Yeah

Reply:
I will leave changing interval to another PR. It would be good to observe the effect of this change first, then we can decide if we want to increase the interval.